src/components/WidgetContactUs/WidgetContactUsData.js

```diff
@@ -1,66 +1,21 @@
 // FPCC
 import { useSiteStore } from 'context/SiteContext'
-import api from 'services/api'
-import { useNotification } from 'context/NotificationContext'
+import { useContactUsSendEmail } from 'common/dataHooks/useContactUs'
 
 function ContactUsData({ widgetData }) {
   const { site } = useSiteStore()
-  const { title, id } = site
-  const { setNotification } = useNotification()
+  const { title } = site
 
   let links = []
   const linkString = widgetData?.settings?.urls || ''
   if (linkString) {
     links = linkString.split(',')
   }
 
-  const submitHandler = async (formData) => {
-    if (formData) {
-      sendMessage(formData)
-    }
-  }
+  const { onSubmit: sendMail } = useContactUsSendEmail()
 
-  const sendMessage = async (formData) => {
-    try {
-      const response = await api.mail.post({
-        siteId: id,
-        name: formData.name,
-        from: formData.email,
-        message: formData.message,
-      })
-
-      if (response?.status === 202) {
-        setNotification({
-          type: 'SUCCESS',
-          message: 'Thank you for emailing the Language Team.',
-        })
-      }
-    } catch (error) {
-      if (error?.name === 'HTTPError') {
-        switch (error?.response?.status) {
-          case 400:
-            setNotification({
-              type: 'ERROR',
-              message: `We've encountered the following error: ${error?.message}.`,
-            })
-            break
-          case 429:
-            setNotification({
-              type: 'ERROR',
-              message:
-                'We have detected potential SPAM issue. Please try again in a few minutes or contact our Help Desk.',
-            })
-            break
-          default:
-            setNotification({
-              type: 'ERROR',
-              message:
-                'We have encountered an unexpected error. Please report this to our Help Desk.',
-            })
-            break
-        }
```

Comment:
Do we still need these error messages somewhere?

Reply:
The `useMutationWithNotification` dataHook will display a message and will also display the error sent from the backend. I believe it will display the following: `"ERROR: Your email was NOT sent. {backend message}. Please try again. If the error persists please contact FirstVoices Support."`

I would be happy to customize the error messages if needed.